tests/test_proxy_functional.py

```diff
@@ -559,15 +559,17 @@ async def test_proxy_https_send_body(
     loop: asyncio.AbstractEventLoop,
 ) -> None:
     sess = aiohttp.ClientSession()
```

Comment:
Shouldn't we just async with here?

Reply:
I didn't change the pattern since we had cases of
```
    finally:
        await sess.close()
        await conn.close()
```

I can if you prefer.